bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaXMLSerializer.java

```diff
@@ -52,20 +51,30 @@
 public class BallerinaXMLSerializer extends OutputStream {
     private static final XMLOutputFactory xmlOutputFactory;
     private static final String XMLNS = "xmlns";
-    private static final String EMPTY_STR = "";
-    public static final String PARSE_XML_OP = "parse xml";
-    public static final String XML = "xml";
-    private static final String XML_NS_URI_PREFIX = "{" + XMLConstants.XML_NS_URI + "}";
     private static final String XML_NAME_SPACE = "http://www.w3.org/XML/1998/namespace";
+    private static final String EMPTY_STR = "";
+    private static final String PARSE_XML_OP = "parse xml";
+    private static final String XML = "xml";
+    private static final String XML_NS_URI_PREFIX = "{" + XMLConstants.XMLNS_ATTRIBUTE_NS_URI + "}";
+    private static final String OUTPUT_VALIDATE_PROPERTY = "com.ctc.wstx.outputValidateStructure";
+    private static final String OUTPUT_FACTORY = "com.ctc.wstx.stax.WstxOutputFactory";
+    private static final String AUTOMATIC_END_ELEMENTS = "com.ctc.wstx.automaticEndElements";
 
     private XMLStreamWriter xmlStreamWriter;
     private Deque<Set<String>> parentNSSet;
     private int nsNumber;
     private boolean withinElement;
+    private static boolean isDefaultFactory = false;
```

Comment:
Since 1.2.x use woodstox do we need this isDefaultFactory change?

Reply:
Reverted the unwanted changes with 81a236285e3e60d48ead606fb808144077009b0c.